sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateAsyncClientTest.java

```diff
@@ -1002,34 +954,11 @@ public void importPemCertificate(HttpClient httpClient, CertificateServiceVersio
 
         importPemCertificateRunner((importCertificateOptions) -> {
             StepVerifier.create(certificateAsyncClient.importCertificate(importCertificateOptions))
-                    .assertNext(importedCertificate -> {
-                        assertEquals(importCertificateOptions.isEnabled(), importedCertificate.getProperties().isEnabled());
-                        assertEquals(CertificateContentType.PEM, importedCertificate.getPolicy().getContentType());
-                    });
+                .assertNext(importedCertificate -> {
+                    assertEquals(importCertificateOptions.isEnabled(), importedCertificate.getProperties().isEnabled());
+                    assertEquals(CertificateContentType.PEM, importedCertificate.getPolicy().getContentType());
+                });
```

Comment:
The recording for this test turned out empty because we are not waiting for the async operation to complete.
```suggestion
                }).verifyComplete();
```